internal/adapters/entrypoints/watcher/pegout_rsk_watcher_test.go

```diff
@@ -476,6 +478,8 @@ func TestPegoutRskDepositWatcher_Start_BlockchainCheck_CheckQuotes(t *testing.T)
 		rskRpc.EXPECT().GetTransactionReceipt(mock.Anything, testRetainedQuote.UserRskTxHash).
 			Return(*receipt, nil).Once()
 		pegoutContract.EXPECT().GetDepositEvents(mock.Anything, uint64(10), mock.MatchedBy(matchUinPtr(20))).Return([]quote.PegoutDeposit{}, nil).Once()
+		btcWallet.On("CreateUnfundedTransactionWithOpReturn", mock.Anything, mock.Anything, mock.Anything).Return([]byte{0x01, 0x02}, nil).Maybe()
```

Comment:
Why maybe? now you should expect this call in all the successful executions

Reply:
I removed this, since the test case is never reaching the point where this is called.